reciprocalspaceship/dataset.py

```diff
@@ -3,7 +3,7 @@
 import pandas as pd
 from pandas.api.types import is_complex_dtype
 
-from reciprocalspaceship import concat, dtypes
+from reciprocalspaceship import concat, dtypes, utils
```

Comment:
Why don't you just import `get_reciprocal_grid_size` in the `utils` function imports below? Wouldn't that be more consistent?